evap/staff/tests/test_tools.py

```diff
@@ -42,8 +42,8 @@ def setUpTestData(cls):
         cls.user1 = mommy.make(UserProfile, username="test1")
         cls.user2 = mommy.make(UserProfile, username="test2")
         cls.user3 = mommy.make(UserProfile, username="test3")
-        cls.group1 = mommy.make(Group, name="group1")
-        cls.group2 = mommy.make(Group, name="group2")
+        cls.group1 = mommy.make(Group, name="group1", pk=4)
+        cls.group2 = mommy.make(Group, name="group2", pk=5)
```

Comment:
any idea why after explicitly setting pks for groups in the migrations, mommy tries to create a group with `pk=1` when not specifying a pk here? this would result in a duplicate key error when running this test.

Reply:
perhaps because django uses some database sequence for choosing the next PK and we're not updating that. bummer. so we can either specify the migration dependencies, instead, or make this test use existing groups (in which case we choose to explicitly specify the pks for all groups in the future, (which doesn't sound too bad to me))

Reply:
okay, then i'll use existing groups for the test.